app/ws/channelHandler.ts

```diff
@@ -23,9 +23,18 @@ const onDisconnect = (userId: string, connectionId: string): void => {
 	}, settings.get('Delay_On_Client_Disconnection') as number);
 };
 
+const addChannel = (channel: string, userId: string): void => {
+	insertChannelToMapping(channel, userId);
+};
+
```

Comment:
it this function really necessary???
it just calls for the insertChannelToMapping

Reply:
You're right, I have change it to 
`const ChannelHandler = { onLogin, onDisconnect, addChannel: insertChannelToMapping, removeUserBindToRoom };`

only for readability, let me know your thoughts,